fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/responses/ViewHelpers.java

```diff
@@ -247,6 +248,25 @@ public String getObjectsAsString(final Graph graph,
     }
 
     /**
+     *
+     * @param graph
+     * @param subject
+     * @return
+     */
+    public int getNumChildren(final Graph graph, final Node subject) {
+        LOGGER.trace("Getting number of children: s:{}, g:{}", subject, graph);
+        final Iterator<Node> iterator = listObjects(graph, subject, HAS_CHILD_COUNT.asNode());
```

Comment:
This looks like a good suggestion; however, it would appear that `Node` needs to be a functional interface to for `.filterKeep(Node::isLiteral)` to compile. Suggestions?
Otherwise, I would suggest moving forward with this impl as is.